.github/workflows/pull-request-teardown.yml

```diff
@@ -32,6 +32,8 @@ jobs:
           helm init --client-only
           echo "$RENKUBOT_KUBECONFIG" > renkubot-kube.config
           helm delete --purge $RENKU_RELEASE
+          helm delete --purge $RENKU_RELEASE-tmp-notebooks || true
+          kubectl delete namespace $RENKU_RELEASE-tmp || true
```

Comment:
why not delete the other namespace as well?

Reply:
good point - if I create the primary namespace manually and not through helm, I should probably also remove it manually...

Reply:
it doesn't matter - helm won't delete the namespace so if you want it gone it needs to be done manually. It wasn't done initially because of RBAC restrictions but now that seems to have been solved.

Reply:
> helm won't delete the namespace

Ah, not even the namespace it created? good to know...